java-api-core/src/com/kinvey/java/sync/SyncManager.java

```diff
@@ -163,7 +163,7 @@ public void executeRequest(final AbstractClient client, SyncRequest request) thr
             e.printStackTrace();
         }
 
-        BaseDataStore networkDataStore = BaseDataStore.collection(request.getCollectionName(), GenericJson.class, StoreType.NETWORK, client);
+        BaseDataStore networkDataStore = BaseDataStore.collection(request.getCollectionName(), GenericJson.class, StoreType.NETWORK, true, client);
```

Comment:
@misha-vakulich This boolean is for the delta set caching flag, right?  If so, should this be set to `true` or `false`?